### PR TITLE
DDF 2490 replace documentation with new outline

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

- Replaces documentation with new outline.
- Removes outdated contents files from old outline.
- Minor cleanup and formatting.

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 
#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@figliold
@kcwire
@lessarderic
@pklinef
@shaundmorris
@stustison

#### How should this be tested?

Build: 
- Builds successfully.
See attached pdf. 
- no missing sections: (search for `unresolved directive` in finished html or pdf.

#### Any background context you want to provide?

Sections 12-13 (Installing, Configuring, Securing) are part of another PR: https://github.com/codice/ddf/pull/1594, so those are lower priority for this PR.

#### What are the relevant tickets?
[DDF-2490](https://codice.atlassian.net/browse/DDF-2490)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/743588/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
@ricklarsen for some reason I'm getting a 404 clicking on the documentation.pdf link. Any ideas?

**Comment by LinkMJB** [comment]
test this please

**Comment by coyotesqrl** [comment]
Likewise on the 404 as of first thing Wednesday morning.

**Comment by ricklarsen** [comment]
upload failure. Try this one: 
[documentation.pdf](https://github.com/codice/ddf/files/745200/documentation.pdf)

And here's a zipped html copy:
[documentation.zip](https://github.com/codice/ddf/files/745211/documentation.zip)

File: distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
+
+To install:
 
-* Navigate to the ${admin-console}.
-* Select *Manage*.
-* Select the ${ddf-geowebcache} application
-* Select the application *Start* button. The application will move to *Active Applications* on startup.
+. Navigate to the ${admin-console}.
+. Select *Manage*.
+. Select the application *Start* button. The application will move to *Active Applications* on startup.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc]: Seems we should still be telling them which application start button to hit. I know admins can infer from context, but we're usually more explicit.

File: distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc [hunk 2]
```diff
@@ -3,58 +3,42 @@ The ${ddf-platform} application is considered to be a core application of the d
...
-* http://karaf.apache.org/[Karaf], 
-* http://cxf.apache.org/CXF[CXF],  and 
-* http://camel.apache.org/[Camel]. 
+* http://karaf.apache.org/[Karaf] 
+* http://cxf.apache.org/CXF[CXF] 
+* http://camel.apache.org/[Camel] 
 
 A Command Scheduler is also included as part of the Platform application.
-The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals.
+The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals for the convenience of a "platform independent" method of running certain commands, such as backing up data or logging settings.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc]: Quoting "platform independent" seems odd to me.

File: distribution/docs/src/main/resources/_contents/_running/command-scheduler-contents.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@ Command Scheduler is a capability exposed through the ${admin-console} (\${secur
 
 ===== Using the Command Scheduler
 
-The <<_command_scheduler,Command Scheduler>> allows administrators to schedule Command Line Shell Commands to be run in a "platform-independent" method.
+The Command Scheduler allows administrators to schedule Command Line Shell Commands to be run in a "platform-independent" method.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_running/command-scheduler-contents.adoc]: Again, the quoted phrase seems odd.

File: distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc]: fixed

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc]: this file is removed in my other pr. I'll edit it there and link to the changes from here.

File: distribution/docs/src/main/resources/_contents/_running/command-scheduler-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/command-scheduler-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 4]
```diff
@@ -92,7 +92,7 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
 .Within the ${branding} console
 ----
 ${branding-lowercase}${at-symbol}local> feature:install wrapper
-${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${ddf-branding-lowercase} -d ${ddf-branding-lowercase} -D "${branding} Service"
+${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${branding-lowercase} -d ${branding-lowercase} -D "${branding} Service"
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: `INSTALLATION_HOME`, `<DDF_HOME>`, and `DDF_INSTALL` are all used in this file. I'm not sure what the convention is, but they all should probably be something like `<DDF_INSTALLATION_HOME>`.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 5]
```diff
@@ -92,57 +92,57 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
...
 +
-.INSTALLATION_HOME/etc/${branding-lowercase}-wrapper.conf
+.<${branding}_HOME>/etc/${branding-lowercase}-wrapper.conf
 [source,java,linenums]
 ----
 #Add the following:
-wrapper.java.additional.10=-D${ddf-branding-lowercase}.home=%KARAF_HOME%
-wrapper.java.additional.12=-Dderby.storage.fileSyncTransactionLog=true
-wrapper.java.additional.11=-server
+wrapper.java.additional.10=-D${branding-lowercase}.home=%KARAF_HOME%
```
Comment (jrnorth) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: I think this is supposed to be `ddf.home` regardless of the distribution you're running.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 6]
```diff
@@ -92,57 +92,57 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
...
-wrapper.java.additional.16=-Djava.util.logging.config.file=%KARAF_HOME%/etc/java.util.logging.properties
-wrapper.java.additional.17=-Dfile.encoding=UTF8
-wrapper.java.additional.18=-XX:+UnlockDiagnosticVMOptions
-wrapper.java.additional.19=-XX:+UnsyncloadClass
-wrapper.java.additional.20=-Dderby.system.home=%KARAF_HOME%/data/derby
-wrapper.java.additional.21=-Dderby.storage.fileSyncTransactionLog=true
+wrapper.java.additional.14=-Djava.security.egd=file:/dev/./urandom
+wrapper.java.additional.15=-Dfile.encoding=UTF8
+wrapper.java.additional.16=-Dkaraf.instances=%KARAF_HOME%/instances
+wrapper.java.additional.17=-D${branding-lowercase}.home=%${branding}_HOME%
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 7]
```diff
@@ -92,57 +92,57 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
...
+wrapper.java.additional.18=-Dkaraf.restart.jvm.supported=true
+wrapper.java.additional.19=-Djava.io.tmpdir=%KARAF_HOME%/data/tmp
+wrapper.java.additional.20=-Djava.util.logging.config.file=%KARAF_HOME%/etc/java.util.logging.properties
+wrapper.java.additional.21=-Dfile.encoding=UTF8
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: removed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 8]
```diff
@@ -92,57 +92,57 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
...
+wrapper.java.additional.22=-XX:+UnlockDiagnosticVMOptions
+wrapper.java.additional.23=-XX:+UnsyncloadClass
+wrapper.java.additional.24=-Dderby.system.home=%KARAF_HOME%/data/derby
+wrapper.java.additional.25=-Dderby.storage.fileSyncTransactionLog=true
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: removed.

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 9]
```diff
@@ -152,33 +152,33 @@ set.default.${branding}_HOME=%KARAF_HOME%
 Run the following command in a console window. The command must be run with elevated permissions.
 +
 ----
-INSTALLATION_HOME/bin/${ddf-branding-lowercase}-service.bat install
+<${branding}_HOME>/bin/${ddf-branding-lowercase}-service.bat install
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: fixed

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 10]
```diff
@@ -152,33 +152,33 @@ set.default.${branding}_HOME=%KARAF_HOME%
...
+root${at-symbol}localhost# chkconfig ${branding-lowercase}-service --add
+root${at-symbol}localhost# chkconfig ${branding-lowercase}-service on
 ----
 +
 *Ubuntu*
 +
 ----
-root${at-symbol}localhost# ln -s ${branding}_INSTALL/bin/${branding-lowercase}-service /etc/init.d/
+root${at-symbol}localhost# ln -s <${branding}_HOME>/bin/${branding-lowercase}-service /etc/init.d/
 root${at-symbol}localhost# update-rc.d -f ${ddf-branding-lowercase}-service defaults
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: fixed.

**Pull request merged**